docs/send-data/collect-from-other-data-sources/azure-blob-storage/troubleshoot-azure-blob-storage-log-collection.md

```diff
@@ -11,18 +11,15 @@ If logs do not start flowing into Sumo Logic after you perform the [Collect Log
 ## Error while deploying the ARM template
 
 If you received the following error while deploying the ARM template: **The Resource 'X' under resource group 'Y' was not found.**
-
-![Azure_ARM-template-error.png](/img/send-data/Azure_ARM-template-error.png)
+<br/><img src={useBaseUrl('/img/send-data/Azure_ARM-template-error.png')}  style={{border: '1px solid gray'}} alt="Azure_ARM-template-error" width="800"/>
 
 If you attempted to redeploy the template after getting this error and used the **same storage account + resource group values** in the first failed deployment, follow these steps.
 
 Refer to [this video](https://sumologic.link/6e020df8d1d7), then redeploy the template, in the following way: 
```

Comment:
this link doesn't work for me

Reply:
@kimsauce This is already updated in the [PR](https://github.com/SumoLogic/sumologic-documentation/pull/3979). PR #3979 will be merged first and this section will be updated accordingly once we Update this branch.